roles/confluent.common/tasks/remove_packages.yml

```
@@ -26,11 +26,19 @@
     msg: "{{ confluent_packages_removed }}"
   when: confluent_packages_removed|length > 0
 
+### Kafka Connect Replicator Binary does not include a service definition by default, thus we need to validate if one exists in order to handle service lifecycle correctly.
+### - ansible_facts.services[service_name + '.service'] is defined
+
+- name: Get Service Facts
+  ansible.builtin.service_facts:
```

We have not made the change to use ansible fully qualified collection names (fqcns) anywhere else in our code yet. This would be the first/only instance of that change.

I do think we should make the switch, but what do you think about creating a jira or gh issue to tackle this change everywhere? That way we wont have two coding patterns going at once in our code base. 

```suggestion
  service_facts:
```

---

Thanks for the feedback, I'll ask @lokesh-shekar to review this further.

---

lgtm. is ansible.builtin. prefix required in this case ? dont think it affects functionality though. good to resolve this if possible.